<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lecteur de diaporama</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="intitule">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Aucun diaporama</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="imgVue">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>155</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnLancer">
          <property name="text">
           <string>&amp;Lancer</string>
          </property>
          <property name="icon">
           <iconset resource="ressourcesDiapo.qrc">
            <normaloff>:/icon/lancerIcon.png</normaloff>:/icon/lancerIcon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnPrecedent">
          <property name="text">
           <string>&amp;Précédent</string>
          </property>
          <property name="icon">
           <iconset resource="ressourcesDiapo.qrc">
            <normaloff>:/icon/precedentIcon.png</normaloff>:/icon/precedentIcon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSuivant">
          <property name="text">
           <string>&amp;Suivant</string>
          </property>
          <property name="icon">
           <iconset resource="ressourcesDiapo.qrc">
            <normaloff>:/icon/suivantIcon.png</normaloff>:/icon/suivantIcon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnArreter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Arrêter</string>
          </property>
          <property name="icon">
           <iconset resource="ressourcesDiapo.qrc">
            <normaloff>:/icon/arreterIcon.png</normaloff>:/icon/arreterIcon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <widget class="QMenu" name="menuParam_trer">
     <property name="title">
      <string>Paramétrer</string>
     </property>
     <property name="icon">
      <iconset resource="ressourcesDiapo.qrc">
       <normaloff>:/icon/paramIcon.png</normaloff>:/icon/paramIcon.png</iconset>
     </property>
     <addaction name="actionVitesse"/>
     <addaction name="actionCharger_diaporama"/>
     <addaction name="actionEnlever_diaporama"/>
     <addaction name="actionMini_lecteur"/>
     <addaction name="actionModifier_titre_image"/>
    </widget>
    <addaction name="action_Quitter"/>
    <addaction name="menuParam_trer"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionA_propos_de">
   <property name="text">
    <string>A propos de...</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionVitesse">
   <property name="icon">
    <iconset resource="ressourcesDiapo.qrc">
     <normaloff>:/icon/speedIcon.png</normaloff>:/icon/speedIcon.png</iconset>
   </property>
   <property name="text">
    <string>Vitesse</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionCharger_diaporama">
   <property name="icon">
    <iconset resource="ressourcesDiapo.qrc">
     <normaloff>:/icon/openIcon.png</normaloff>:/icon/openIcon.png</iconset>
   </property>
   <property name="text">
    <string>Charger diaporama</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnlever_diaporama">
   <property name="icon">
    <iconset resource="ressourcesDiapo.qrc">
     <normaloff>:/icon/removeIcon.png</normaloff>:/icon/removeIcon.png</iconset>
   </property>
   <property name="text">
    <string>Enlever diaporama</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionMini_lecteur">
   <property name="text">
    <string>Mini-lecteur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionModifier_titre_image">
   <property name="text">
    <string>Modifier titre image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressourcesDiapo.qrc"/>
 </resources>
 <connections/>
</ui>
